### PR TITLE
Leerzeichen nach Circumflex einfügen

### LINKED ISSUE #10: Leerzeichen nach dem return-Operator
Als Squeakentwickler möchte ich ein Whitespace nach ^  *(bsp.: ^ self)* einführen, um den Rückgabewert leicht erkennbar zu machen und gut abzugrenzen.

5

### PR DESCRIPTION
**Pull request by PatriciaSowa**
Closes #10

### CODE REVIEW

File: packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st [hunk 1]
```diff
@@ -0,0 +1,9 @@
+actions
+FinalStatement: aNode with: aTerminal with: aStatement with: anotherTerminal
+	
+	| circumflex |
+	circumflex := aTerminal children first.
+	circumflex isNil
+		ifTrue: [^ self value: aStatement]
+		ifFalse: [^ circumflex, ' ', (self value: aStatement)]
```
Comment (T4rikA) [inline packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st]: space between circumflex and , missing

**Review by felixauringer — APPROVED** [review]

File: packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st [hunk 2]
```diff
@@ -0,0 +1,9 @@
+actions
+FinalStatement: aNode with: aTerminal with: aStatement with: anotherTerminal
```
Comment (T4rikA) [inline packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st]: should be changed to with:and:and: or be consistent in every method with:with:with:

**Review by T4rikA — CHANGES_REQUESTED** [review]
request changes in FinalStatement:with:with:with:

**Review by Paula-Kli — COMMENTED** [review]
Tarik has some change requests

File: packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st (hunk 2 above)
Comment (Paula-Kli) [reply packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st]: I think I prefer reading with:and:and:

**Review by felixauringer — APPROVED** [review]

**Review by T4rikA — APPROVED** [review]

File: packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st (hunk 2 above)
Comment (felixauringer) [reply packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st]: We changed it to with:and:and:

File: packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st (hunk 1 above)
Comment (felixauringer) [reply packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st]: We added spaces around the commas.

**Review by felixauringer — APPROVED** [review]

File: packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st (hunk 1 above)
Comment (felixauringer) [reply packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st]: I added spaces in the blocks.

**Review by Paula-Kli — APPROVED** [review]

**Pull request merged**